shared/models.go

```diff
@@ -162,18 +162,18 @@ type Manifest struct {
 }
 
 // FilterByPath filters all the manifest items by path.
-func (m Manifest) FilterByPath(paths mapset.Set) (result Manifest, err error) {
+func (m Manifest) FilterByPath(paths ...string) (result Manifest, err error) {
```

Comment:
Is there a caller of this function that actually needs to pass a variable length argument list? It seems to me the caller unpacks a slice using trailing `...` and the function packs it up again using leading `...`.

Reply:
No, but I'd doubt that this leads to any performance hit

Reply:
I wasn't concerned about performance, but the ergonomics of the API.